libsrc/_DEVELOPMENT/stdlib/z80/asm_atoi.asm

```diff
@@ -32,12 +32,12 @@ asm_atoi:
 asm0_atoi:
```

Comment:
The classic 8080 uses the atoi that's in libsrc/stdlib/8080/asm_atoi

If the newlib one has now been adapted we can switch over.